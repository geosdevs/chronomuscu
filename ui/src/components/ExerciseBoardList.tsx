import { ExerciseBoardData } from "../App";
import ExerciseBoard from "./ExerciseBoard/ExerciseBoard";

type ExerciseBoardListProps = {
  exerciseBoards: ExerciseBoardData[]
  activeBoardId: number
  onNextExerciseClick: Function
};

export default function ExerciseBoardList({
  exerciseBoards,
  activeBoardId,
  onNextExerciseClick
}: ExerciseBoardListProps) {
  return (
    <>
      {exerciseBoards.map((exerciseBoard) => (
        <ExerciseBoard
          key={`exercise-board-${exerciseBoard.id}`}
          isActive={activeBoardId === exerciseBoard.id}
          boardData={exerciseBoard}
          onNextExerciseClick={onNextExerciseClick}
        ></ExerciseBoard>
      ))}
    </>
  );
}
