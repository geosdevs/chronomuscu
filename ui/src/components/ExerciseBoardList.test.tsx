import renderer from "react-test-renderer";
import ExerciseBoardList from "./ExerciseBoardList";
import { ExerciseBoardData } from "../app-types";

describe("ExerciseBoardList", () => {
  it("should render", () => {
    const handleExerciseNameChangeMock = jest.fn();
    const onNextExerciseClickMock = jest.fn();
    const exerciseBoardsData: ExerciseBoardData[] = [
      { id: 34, exerciseName: "Squat" },
      { id: 35, exerciseName: "Deadlift" },
      { id: 36, exerciseName: "Dragon flag" },
    ];
    const component = renderer.create(
      <ExerciseBoardList
        activeBoardId={7}
        exerciseBoards={exerciseBoardsData}
        handleExerciseNameChange={handleExerciseNameChangeMock}
        onNextExerciseClick={onNextExerciseClickMock}
      />
    );
    let jsonTree = component.toJSON();

    expect(jsonTree).toMatchSnapshot();
    expect(handleExerciseNameChangeMock).not.toHaveBeenCalled();
    expect(onNextExerciseClickMock).not.toHaveBeenCalled();
  });
});
