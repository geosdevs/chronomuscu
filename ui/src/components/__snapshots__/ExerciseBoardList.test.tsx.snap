// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExerciseBoardList should render 1`] = `
Array [
  <div
    className="hidden"
  >
    <div
      className="my-2 p-2 md:pl-4"
    >
      <div
        className="flex"
      >
        <div>
          <h2
            className="text-2xl hover:scale-105 duration-300"
          >
            <strong
              className="text-gunmetal"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-hashtag fa-lg "
                data-icon="hashtag"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M181.3 32.4c17.4 2.9 29.2 19.4 26.3 36.8L197.8 128h95.1l11.5-69.3c2.9-17.4 19.4-29.2 36.8-26.3s29.2 19.4 26.3 36.8L357.8 128H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H347.1L325.8 320H384c17.7 0 32 14.3 32 32s-14.3 32-32 32H315.1l-11.5 69.3c-2.9 17.4-19.4 29.2-36.8 26.3s-29.2-19.4-26.3-36.8l9.8-58.7H155.1l-11.5 69.3c-2.9 17.4-19.4 29.2-36.8 26.3s-29.2-19.4-26.3-36.8L90.2 384H32c-17.7 0-32-14.3-32-32s14.3-32 32-32h68.9l21.3-128H64c-17.7 0-32-14.3-32-32s14.3-32 32-32h68.9l11.5-69.3c2.9-17.4 19.4-29.2 36.8-26.3zM187.1 192L165.8 320h95.1l21.3-128H187.1z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
              34
            </strong>
             
            <span
              className="text-cordovan hover:text-ecru duration-300"
              onClick={[Function]}
            >
              Squat
            </span>
          </h2>
        </div>
        <a
          className="group ml-6 relative inline-block text-sm font-bold text-chinarose hover:text-ecru focus:outline-none active:text-slate-500 transition-colors duration-200"
          href="/"
          onClick={[Function]}
        >
          <span
            className="rounded-md absolute inset-0 translate-x-0 translate-y-0 bg-ecru transition-transform group-hover:translate-x-0.5 group-hover:translate-y-0.5"
          />
          <span
            className="rounded-md relative block border border-chinarose hover:border-current bg-white px-2.5 py-1.5"
          >
             
            Next 
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-forward "
              data-icon="forward"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M52.5 440.6c-9.5 7.9-22.8 9.7-34.1 4.4S0 428.4 0 416V96C0 83.6 7.2 72.3 18.4 67s24.5-3.6 34.1 4.4L224 214.3V256v41.7L52.5 440.6zM256 352V256 128 96c0-12.4 7.2-23.7 18.4-29s24.5-3.6 34.1 4.4l192 160c7.3 6.1 11.5 15.1 11.5 24.6s-4.2 18.5-11.5 24.6l-192 160c-9.5 7.9-22.8 9.7-34.1 4.4s-18.4-16.6-18.4-29V352z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
             
          </span>
        </a>
      </div>
      <div
        className="md:w-fit mx-2 my-4 grid"
      >
        <div
          className="row-span-1 col-span-4 row-start-1 w-100"
        >
          <div
            className="flex rounded-xl bg-white p-4 ring ring-indigo-50 sm:p-6 lg:p-8"
          >
            <div
              className="my-2 w-4/5"
            >
              <div
                className="text-center"
              >
                <span
                  className="font-bold text-5xl text-frenchgray"
                >
                  0
                </span>
              </div>
              <span
                aria-labelledby="ProgressLabel"
                className="block rounded-full my-2 bg-gray-200"
                role="progressbar"
              >
                <span
                  className="block h-3 rounded-full"
                  style={
                    Object {
                      "width": "0%",
                    }
                  }
                />
              </span>
              <span
                className="ml-1 mt-1.5 inline-block"
              >
                <span
                  className="inline-flex items-center justify-center rounded-full px-2.5 py-0.5 bg-white text-gunmetal"
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-check fa-lg "
                    data-icon="check"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                  <p
                    className="whitespace-nowrap ml-1 text-sm"
                  >
                    Done
                  </p>
                </span>
              </span>
            </div>
            <span
              className="text-3xl play-toolbar-flex inline-flex justify-center overflow-hidden rounded-md bg-white my-2 mx-0 md:mx-4"
            >
              <button
                className="mb-1 inline-block p-3 text-gray-700 hover:text-ecru hover:scale-110 duration-300 focus:relative disabled:text-frenchgray"
                disabled={true}
                onClick={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-play "
                  data-icon="play"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M73 39c-14.8-9.1-33.4-9.4-48.5-.9S0 62.6 0 80V432c0 17.4 9.4 33.4 24.5 41.9s33.7 8.1 48.5-.9L361 297c14.3-8.7 23-24.2 23-41s-8.7-32.2-23-41L73 39z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </button>
              <button
                className="inline-block p-3 text-chinarose enabled:hover:scale-110 hover:text-ecru duration-300 focus:relative disabled:text-frenchgray text-frenchgray"
                disabled={true}
                onClick={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-stop "
                  data-icon="stop"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 128C0 92.7 28.7 64 64 64H320c35.3 0 64 28.7 64 64V384c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V128z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </button>
            </span>
          </div>
        </div>
        <div
          className="w-fit my-2 md:mx-2 row-span-2 col-span-4 row-start-3 md:col-span-2 md:row-start-2"
        >
          <div
            className="hidden"
          >
            <span>
              Goal
            </span>
            <span>
              4*20
            </span>
          </div>
          <div
            className=""
          >
            <h3
              className="text-2xl font-bold text-gray-900"
            >
              Current set history
            </h3>
            <div
              className="grid"
            />
          </div>
        </div>
        <div
          className="my-2 row-start-2 col-span-4 row-span-1 overflow-x-auto w-100 md:row-span-2 md:col-span-2 md:col-start-3"
        >
          <div
            className="flex flex-row md:flex-wrap md:w-fit"
          />
        </div>
      </div>
    </div>
  </div>,
  <div
    className="hidden"
  >
    <div
      className="my-2 p-2 md:pl-4"
    >
      <div
        className="flex"
      >
        <div>
          <h2
            className="text-2xl hover:scale-105 duration-300"
          >
            <strong
              className="text-gunmetal"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-hashtag fa-lg "
                data-icon="hashtag"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M181.3 32.4c17.4 2.9 29.2 19.4 26.3 36.8L197.8 128h95.1l11.5-69.3c2.9-17.4 19.4-29.2 36.8-26.3s29.2 19.4 26.3 36.8L357.8 128H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H347.1L325.8 320H384c17.7 0 32 14.3 32 32s-14.3 32-32 32H315.1l-11.5 69.3c-2.9 17.4-19.4 29.2-36.8 26.3s-29.2-19.4-26.3-36.8l9.8-58.7H155.1l-11.5 69.3c-2.9 17.4-19.4 29.2-36.8 26.3s-29.2-19.4-26.3-36.8L90.2 384H32c-17.7 0-32-14.3-32-32s14.3-32 32-32h68.9l21.3-128H64c-17.7 0-32-14.3-32-32s14.3-32 32-32h68.9l11.5-69.3c2.9-17.4 19.4-29.2 36.8-26.3zM187.1 192L165.8 320h95.1l21.3-128H187.1z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
              35
            </strong>
             
            <span
              className="text-cordovan hover:text-ecru duration-300"
              onClick={[Function]}
            >
              Deadlift
            </span>
          </h2>
        </div>
        <a
          className="group ml-6 relative inline-block text-sm font-bold text-chinarose hover:text-ecru focus:outline-none active:text-slate-500 transition-colors duration-200"
          href="/"
          onClick={[Function]}
        >
          <span
            className="rounded-md absolute inset-0 translate-x-0 translate-y-0 bg-ecru transition-transform group-hover:translate-x-0.5 group-hover:translate-y-0.5"
          />
          <span
            className="rounded-md relative block border border-chinarose hover:border-current bg-white px-2.5 py-1.5"
          >
             
            Next 
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-forward "
              data-icon="forward"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M52.5 440.6c-9.5 7.9-22.8 9.7-34.1 4.4S0 428.4 0 416V96C0 83.6 7.2 72.3 18.4 67s24.5-3.6 34.1 4.4L224 214.3V256v41.7L52.5 440.6zM256 352V256 128 96c0-12.4 7.2-23.7 18.4-29s24.5-3.6 34.1 4.4l192 160c7.3 6.1 11.5 15.1 11.5 24.6s-4.2 18.5-11.5 24.6l-192 160c-9.5 7.9-22.8 9.7-34.1 4.4s-18.4-16.6-18.4-29V352z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
             
          </span>
        </a>
      </div>
      <div
        className="md:w-fit mx-2 my-4 grid"
      >
        <div
          className="row-span-1 col-span-4 row-start-1 w-100"
        >
          <div
            className="flex rounded-xl bg-white p-4 ring ring-indigo-50 sm:p-6 lg:p-8"
          >
            <div
              className="my-2 w-4/5"
            >
              <div
                className="text-center"
              >
                <span
                  className="font-bold text-5xl text-frenchgray"
                >
                  0
                </span>
              </div>
              <span
                aria-labelledby="ProgressLabel"
                className="block rounded-full my-2 bg-gray-200"
                role="progressbar"
              >
                <span
                  className="block h-3 rounded-full"
                  style={
                    Object {
                      "width": "0%",
                    }
                  }
                />
              </span>
              <span
                className="ml-1 mt-1.5 inline-block"
              >
                <span
                  className="inline-flex items-center justify-center rounded-full px-2.5 py-0.5 bg-white text-gunmetal"
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-check fa-lg "
                    data-icon="check"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                  <p
                    className="whitespace-nowrap ml-1 text-sm"
                  >
                    Done
                  </p>
                </span>
              </span>
            </div>
            <span
              className="text-3xl play-toolbar-flex inline-flex justify-center overflow-hidden rounded-md bg-white my-2 mx-0 md:mx-4"
            >
              <button
                className="mb-1 inline-block p-3 text-gray-700 hover:text-ecru hover:scale-110 duration-300 focus:relative disabled:text-frenchgray"
                disabled={true}
                onClick={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-play "
                  data-icon="play"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M73 39c-14.8-9.1-33.4-9.4-48.5-.9S0 62.6 0 80V432c0 17.4 9.4 33.4 24.5 41.9s33.7 8.1 48.5-.9L361 297c14.3-8.7 23-24.2 23-41s-8.7-32.2-23-41L73 39z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </button>
              <button
                className="inline-block p-3 text-chinarose enabled:hover:scale-110 hover:text-ecru duration-300 focus:relative disabled:text-frenchgray text-frenchgray"
                disabled={true}
                onClick={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-stop "
                  data-icon="stop"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 128C0 92.7 28.7 64 64 64H320c35.3 0 64 28.7 64 64V384c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V128z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </button>
            </span>
          </div>
        </div>
        <div
          className="w-fit my-2 md:mx-2 row-span-2 col-span-4 row-start-3 md:col-span-2 md:row-start-2"
        >
          <div
            className="hidden"
          >
            <span>
              Goal
            </span>
            <span>
              4*20
            </span>
          </div>
          <div
            className=""
          >
            <h3
              className="text-2xl font-bold text-gray-900"
            >
              Current set history
            </h3>
            <div
              className="grid"
            />
          </div>
        </div>
        <div
          className="my-2 row-start-2 col-span-4 row-span-1 overflow-x-auto w-100 md:row-span-2 md:col-span-2 md:col-start-3"
        >
          <div
            className="flex flex-row md:flex-wrap md:w-fit"
          />
        </div>
      </div>
    </div>
  </div>,
  <div
    className="hidden"
  >
    <div
      className="my-2 p-2 md:pl-4"
    >
      <div
        className="flex"
      >
        <div>
          <h2
            className="text-2xl hover:scale-105 duration-300"
          >
            <strong
              className="text-gunmetal"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-hashtag fa-lg "
                data-icon="hashtag"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M181.3 32.4c17.4 2.9 29.2 19.4 26.3 36.8L197.8 128h95.1l11.5-69.3c2.9-17.4 19.4-29.2 36.8-26.3s29.2 19.4 26.3 36.8L357.8 128H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H347.1L325.8 320H384c17.7 0 32 14.3 32 32s-14.3 32-32 32H315.1l-11.5 69.3c-2.9 17.4-19.4 29.2-36.8 26.3s-29.2-19.4-26.3-36.8l9.8-58.7H155.1l-11.5 69.3c-2.9 17.4-19.4 29.2-36.8 26.3s-29.2-19.4-26.3-36.8L90.2 384H32c-17.7 0-32-14.3-32-32s14.3-32 32-32h68.9l21.3-128H64c-17.7 0-32-14.3-32-32s14.3-32 32-32h68.9l11.5-69.3c2.9-17.4 19.4-29.2 36.8-26.3zM187.1 192L165.8 320h95.1l21.3-128H187.1z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
              36
            </strong>
             
            <span
              className="text-cordovan hover:text-ecru duration-300"
              onClick={[Function]}
            >
              Dragon flag
            </span>
          </h2>
        </div>
        <a
          className="group ml-6 relative inline-block text-sm font-bold text-chinarose hover:text-ecru focus:outline-none active:text-slate-500 transition-colors duration-200"
          href="/"
          onClick={[Function]}
        >
          <span
            className="rounded-md absolute inset-0 translate-x-0 translate-y-0 bg-ecru transition-transform group-hover:translate-x-0.5 group-hover:translate-y-0.5"
          />
          <span
            className="rounded-md relative block border border-chinarose hover:border-current bg-white px-2.5 py-1.5"
          >
             
            Next 
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-forward "
              data-icon="forward"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M52.5 440.6c-9.5 7.9-22.8 9.7-34.1 4.4S0 428.4 0 416V96C0 83.6 7.2 72.3 18.4 67s24.5-3.6 34.1 4.4L224 214.3V256v41.7L52.5 440.6zM256 352V256 128 96c0-12.4 7.2-23.7 18.4-29s24.5-3.6 34.1 4.4l192 160c7.3 6.1 11.5 15.1 11.5 24.6s-4.2 18.5-11.5 24.6l-192 160c-9.5 7.9-22.8 9.7-34.1 4.4s-18.4-16.6-18.4-29V352z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
             
          </span>
        </a>
      </div>
      <div
        className="md:w-fit mx-2 my-4 grid"
      >
        <div
          className="row-span-1 col-span-4 row-start-1 w-100"
        >
          <div
            className="flex rounded-xl bg-white p-4 ring ring-indigo-50 sm:p-6 lg:p-8"
          >
            <div
              className="my-2 w-4/5"
            >
              <div
                className="text-center"
              >
                <span
                  className="font-bold text-5xl text-frenchgray"
                >
                  0
                </span>
              </div>
              <span
                aria-labelledby="ProgressLabel"
                className="block rounded-full my-2 bg-gray-200"
                role="progressbar"
              >
                <span
                  className="block h-3 rounded-full"
                  style={
                    Object {
                      "width": "0%",
                    }
                  }
                />
              </span>
              <span
                className="ml-1 mt-1.5 inline-block"
              >
                <span
                  className="inline-flex items-center justify-center rounded-full px-2.5 py-0.5 bg-white text-gunmetal"
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-stop fa-lg "
                    data-icon="stop"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 384 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 128C0 92.7 28.7 64 64 64H320c35.3 0 64 28.7 64 64V384c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V128z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                  <p
                    className="whitespace-nowrap ml-1 text-sm"
                  >
                    Stopped
                  </p>
                </span>
              </span>
            </div>
            <span
              className="text-3xl play-toolbar-flex inline-flex justify-center overflow-hidden rounded-md bg-white my-2 mx-0 md:mx-4"
            >
              <button
                className="mb-1 inline-block p-3 text-gray-700 hover:text-ecru hover:scale-110 duration-300 focus:relative disabled:text-frenchgray"
                disabled={false}
                onClick={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-play "
                  data-icon="play"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M73 39c-14.8-9.1-33.4-9.4-48.5-.9S0 62.6 0 80V432c0 17.4 9.4 33.4 24.5 41.9s33.7 8.1 48.5-.9L361 297c14.3-8.7 23-24.2 23-41s-8.7-32.2-23-41L73 39z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </button>
              <button
                className="inline-block p-3 text-chinarose enabled:hover:scale-110 hover:text-ecru duration-300 focus:relative disabled:text-frenchgray"
                disabled={true}
                onClick={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-stop "
                  data-icon="stop"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 128C0 92.7 28.7 64 64 64H320c35.3 0 64 28.7 64 64V384c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V128z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </button>
            </span>
          </div>
        </div>
        <div
          className="w-fit my-2 md:mx-2 row-span-2 col-span-4 row-start-3 md:col-span-2 md:row-start-2"
        >
          <div
            className="hidden"
          >
            <span>
              Goal
            </span>
            <span>
              4*20
            </span>
          </div>
          <div
            className=""
          >
            <h3
              className="text-2xl font-bold text-gray-900"
            >
              Current set history
            </h3>
            <div
              className="grid"
            />
          </div>
        </div>
        <div
          className="my-2 row-start-2 col-span-4 row-span-1 overflow-x-auto w-100 md:row-span-2 md:col-span-2 md:col-start-3"
        >
          <div
            className="flex flex-row md:flex-wrap md:w-fit"
          />
        </div>
      </div>
    </div>
  </div>,
]
`;
